src/authoring/BehaveToAuthor.ts

```diff
@@ -85,5 +99,75 @@ export const behaveToAuthor = (graph: string): [Node[], Edge[], ICustomEvent[],
     id++;
   });
 
+  if (needsReflow) {
```

Comment:
if this works better than the current auto ordering then lets switch to this one, up to you